CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 2.1.0 / 2018-1-19
+
+[CHANGE] Remote read `read_recent` option is not false by default.
+[FEATURE] New Service Discovery UI showing labels before and after relabelling.
+[FEATURE] New Admin APIs added to v1 to delete, snapshot and cleanup tombstones.
```

Comment:
"delete, snapshot, and clean up tombstones."

(I was like, "What's a cleanup tombstone?")